packages/crayons-core/src/components/radio/radio.scss

```diff
@@ -59,12 +59,12 @@ input[type='radio'] {
   & + label {
     user-select: none;
     cursor: pointer;
-    margin-bottom: 0;
+    margin-bottom: 4px;
     vertical-align: middle;
     font-size: 14px;
     line-height: 20px;
     color: #12344d;
-    font-weight: 500;
+    font-weight: 600;
```

Comment:
There was no difference mentioned for font-weights for title and sub-title in DSM , using the same values as mentioned for checkbox